packages/react-aria-components/src/Slider.tsx

```diff
@@ -63,7 +80,13 @@ function Slider<T extends number | number[]>(props: SliderProps<T>, ref: Forward
 
   let renderProps = useRenderProps({
     ...props,
-    values: state,
+    values: {
+      orientation: state.orientation,
+      isDisabled: state.isDisabled,
+      isFocused,
+      isFocusVisible,
+      state
```

Comment:
should this state (and the states in other components) be spread instead?

Reply:
I think we decided to include the state object during a discussion in slack. I think this also gives us a pretty clear delimiter for what we should include as a data attribute.

Reply:
is it weird though to have `isDisabled: state.isDisabled` but then have the same information available in the state object separately? Might be confusing to the user as to if there is a difference between `{({state, isDisabled}) => use state.isDisabled or isDisabled`.

Reply:
I'm fine with this if we are treating the `state` object as a special additional info source, we should make that clear in the docs if so. Also looks like the Slider docs examples will need to be updated to extract state from the set of renderProps